### PR TITLE
create separate groups of packages

### LINKED ISSUE #3800: Minimize number of Renovate bot pull requests
Renovate bot opens a separate PR for each dependency upgrade, which creates a lot of PRs and takes a lot of CI time. We can reduce Renovate frequency using scheduling and package groups, see https://renovatebot.com/docs/noise-reduction/

### PR DESCRIPTION
**Pull request by bahmutov**
- closes #3800 

- all dependencies by default updates once per month
- defined several groups that update weekly
- Gleb's dependencies and `@cypress` dependencies are grouped together and updated nightly

### CODE REVIEW

File: renovate.json [hunk 1]
```diff
@@ -11,11 +11,50 @@
...
+      "schedule": "before 3am on Monday"
+    },
+    {
+      "packagePatterns": "^eslint",
+      "groupName": "eslint",
+      "schedule": "before 3am on Monday"
+    },
+    {
+      "packageNames": ["typescript", "ts-node", "tslint-config-standard"],
+      "groupName": "typescript"
```
Comment (bahmutov) [inline renovate.json]: this group will have all those dependencies and will follow common schedule

File: renovate.json [hunk 2]
```diff
@@ -11,11 +11,50 @@
   },
   "rangeStrategy": "pin",
   "packageRules": [
+    {
+      "packagePatterns": "^@cypress",
```
Comment (bahmutov) [inline renovate.json]: all package names that start with `@cypress` will be in 1 PR created nightly

File: renovate.json [hunk 3]
```diff
@@ -11,11 +11,50 @@
...
       ],
+      "groupName": "Gleb's NPM packages",
+      "schedule": "after 2am and before 4am"
+    },
+    {
+      "packageNames": "node",
       "enabled": false
+    },
+    {
+      "packagePatterns": "^@types",
```
Comment (bahmutov) [inline renovate.json]: all types will be in one PR weekly

**Comment by brian-mann** [comment]
This looks great

**Comment by bahmutov** [comment]
ok, will merge and will see what Renovate does, should minimize the noise - and we can add groups as needed

**Pull request merged**